src/SharedFrameworkValidation/RestoreSDKProject/RestoreSDKProject.csproj

```diff
@@ -0,0 +1,64 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RuntimeFrameworkVersion>$(_RuntimeFrameworkVersion)</RuntimeFrameworkVersion>
+    <CoreFxPackageVersion>4.4.0-$(CoreFxExpectedPrerelease)</CoreFxPackageVersion>
+    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CSharp" Version="$(CoreFxPackageVersion)" />
```

Comment:
For the first iteration, I just added all of these manually basing myself of from [here](https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/Last_Build_Packages.txt). I plan in the future automating the process by looking into that list from dotnet/versions and filtering out the runtime-specific packages + the private metapackages.